README.md

```
@@ -1,60 +1,249 @@
-Aethelgard: Echoes of the Shard
-Ein Singleplayer- und Koop-Action-RPG, entwickelt mit der Godot Engine. Inspiriert von Genre-Größen wie Diablo und Path of Exile, fokussiert auf tiefgründige Charakterentwicklung und endlose Wiederspielbarkeit.
+<div align="center">
 
-📖 Über das Spiel
-"Aethelgard: Echoes of the Shard" ist die Antwort auf den Wunsch nach einem tiefgründigen ARPG-Erlebnis, das intensive Kämpfe mit strategischer Charakter-Meisterschaft verbindet. Es ist als fokussiertes Singleplayer-Erlebnis mit optionalem Koop-Modus konzipiert. Das Spiel ist in zwei Phasen unterteilt:
+<img src="docs/assets/readme_banner.jpg" alt="Aethelgard: Echoes of the Shard — Banner" width="100%">
 
-Die Kampagne: Eine fesselnde, handlungsgetriebene Reise, die dich in die zersplitterte Welt von Aethelgard und die Kernmechaniken des Spiels einführt.
+# Aethelgard
+### *Echoes of the Shard*
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)